docs/src/pages/layout/breakpoints/breakpoints.md

```diff
@@ -48,10 +48,30 @@ export default withWidth()(MyComponent);
 
 In the following demo, we change the rendered DOM element (*em*, <u>u</u>, ~~del~~ & span) based on the screen width.
 
-{{"demo": "pages/layout/breakpoints/WithWidth.js"}}
+{{"demo": "pages/layout/breakpoints/WithWidth.js" }}
 
 ⚠️ `withWidth()` server-side rendering support is limited.
 
+## useWidth()
+
+> ⚠️ `useWidth` is experimental as Hooks aren't stable yet, therefore it is exported with an unstable prefix\
+__PLEASE NOTE: `useWidth` depends on react >= 16.7.0-alpha.0, react-dom >= 16.7.0-alpha.0 and @material-ui/styles.__
+
+```js
```

Comment:
jsx

Reply:
I thought so, I copied the other examples on the page. Does this mean the withWidth example should be jsx too?

Reply:
We are using jsx when the JSX syntax is present and JS when only JavaScript is used.

Reply:
https://github.com/mui-org/material-ui/master/docs/src/pages/layout/breakpoints/breakpoints.md#L39-L47
?